docs/release.md

```diff
@@ -0,0 +1,149 @@
+# steps
+
+1. Sync latest master
+1. new branch
+1. Sync the downstream pipeline repo
+1. Checkout release branch
+1. Copy the release yaml from the pipelines repo to operator
```

Comment:
shall we make it
 Copy the release yaml from `downstream-repo/openshift/release/tektoncd-pipeline-v0.7.0.yaml`

Reply:
if you include version, the doc will need to update every time.. Lets keep the doc generic and refine it in the next iteration.